plugins/destination/postgresql/client/spec.go

```diff
@@ -8,8 +8,8 @@ import (
 
 const (
 	defaultBatchSize      = 10000
-	defaultBatchSizeBytes = 1000000
-	defaultBatchTimeout   = 10 * time.Second
+	defaultBatchSizeBytes = 100000000
```

Comment:
We should release a new Postgres version with this

Reply:
I made https://github.com/cloudquery/cloudquery/releases/tag/plugins-destination-postgresql-v6.0.1 a pre-release since it has the old batch settings and also doesn't have https://github.com/cloudquery/cloudquery/pull/13332